_includes/modern-appendix.md

```diff
@@ -438,24 +446,37 @@ Sent as a reply to the [`WHOWAS`](#whowas-message) command, this numeric shows d
 
 ### `RPL_WHOISIDLE (317)`
 
-      "<client> <nick> <secs> [<signon>] :seconds idle, signon time
+      "<client> <nick> <secs> <signon> :seconds idle, signon time
 
 Sent as a reply to the [`WHOIS`](#whois-message) command, this numeric indicates how long the client with the nickname `<nick>` has been idle. `<secs>` is the number of seconds since the client has been active. Servers generally denote specific commands (for instance, perhaps [`JOIN`](#join-message), [`PRIVMSG`](#privmsg-message), [`NOTICE`](#notice-message), etc) as updating the 'idle time', and calculate this off when the idle time was last updated. `<signon>` is a unix timestamp representing when the user joined the network. The text used in the last param of this message may vary.
 
 ### `RPL_ENDOFWHOIS (318)`
 
       "<client> <nick> :End of /WHOIS list"
 
-Sent as a reply to the [`WHOIS`](#whois-message) command, this numeric indicates the end of a `WHOIS` response for the client with the nickname `<nick>`. This numeric is sent after all other `WHOIS` response numerics have been sent to the client.
+Sent as a reply to the [`WHOIS`](#whois-message) command, this numeric indicates the end of a `WHOIS` response for the client with the nickname `<nick>`.
+
+`<nick>` MUST be exactly the `<nick>` parameter sent by the client in its `WHOIS` message.
+This means the case MUST be preserved, and if the client sent multiple nicks, this MUST be the comma-separated list of nicks, even if some of them were dropped.
+
+This numeric is sent after all other `WHOIS` response numerics have been sent to the client.
 
 ### `RPL_WHOISCHANNELS (319)`
 
       "<client> <nick> :[prefix]<channel>{ [prefix]<channel>}
 
 Sent as a reply to the [`WHOIS`](#whois-message) command, this numeric lists the channels that the client with the nickname `<nick>` is joined to and their status in these channels. `<prefix>` is the highest [channel membership prefix](#channel-membership-prefixes) that the client has in that channel, if the client has one. `<channel>` is the name of a channel that the client is joined to. The last parameter of this numeric is a list of `[prefix]<channel>` pairs, delimited by a SPACE character `(' ', 0x20)`.
 
+`RPL_WHOISCHANNELS` can be sent multiple times in the same whois reply, if the target is too many channels to fit in a single message.
```

Comment:
```suggestion
`RPL_WHOISCHANNELS` can be sent multiple times in the same whois reply, if the target is joined to too many channels to fit in a single message.
```

Reply:
Actually I think I meant "if the target is on too many channels", which sounds better IMO